docs/vspec.md

```diff
@@ -306,12 +310,16 @@ Warning: Attribute(s) quality, source in element Speed not a core or known exten
 You asked for strict checking. Terminating.
 ```
 
-You can whitelist extended metadata attributes using the `-e` parameter:
+You can whitelist extended metadata attributes using the `-e` / `--extended-attributes` argument.
 
 ```bash
-vspec export json -I spec spec/VehicleSignalSpecification.vspec -e quality -e source -l overlay.vspec --output test.json
+vspec export json -I spec -s spec/VehicleSignalSpecification.vspec -e quality -e source -l overlay.vspec -o test.json
+vspec export json -I spec --vspec spec/VehicleSignalSpecification.vspec --extended-attributes quality --extended-attributes source --overlays overlay.vspec --output test.json
 ```
 
+*Note: A comma separated list can no longer be used to specify extended attributes!*
```

Comment:
maybe use an [alert](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts)